i18n/de/i18n.json

```diff
@@ -0,0 +1,106 @@
+{
+  "index": {
+    "title": "Lizenzhinweisgenerator",
+    "description": "Lizenzhinweise für Bilder aus Wikipedia und Wikimedia Commons",
+    "no-script": "Sie müssen JavaScript zur Nutzung dieser Website aktiviert.",
```

Comment:
I am guessing this is not entirely correct Kraut speak, although it contains all words I'd expect: "müssen", "Javascript", "aktiviert" :)
Let's have @jakobw and @WMDE-Fisch suggest something better